src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -330,6 +331,47 @@ private boolean areEqualIgnoringCase(CharSequence actual, CharSequence expected)
   }
 
   /**
+   * Verifies that two {@code CharSequence}s are equal, ignoring any changes in whitespace.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the actual {@code CharSequence}.
+   * @param expected the expected {@code CharSequence}.
+   * @throws AssertionError if the given {@code CharSequence}s are not equal.
+   */
+  public void assertEqualsIgnoringWhitespace(AssertionInfo info, CharSequence actual, CharSequence expected) {
+      if (!areEqualIgnoringWhitespace(actual, expected)) {
```

Comment:
should check that `actual` is not null with `assertNotNull(info, actual);` (I know it's missing in `assertEqualsIgnoringCase`).